ocaml/database/db_cache_types.ml

```diff
@@ -144,7 +145,7 @@ module Table = struct
     let rec loop xs acc = match xs with
       | (created,deleted,r)::xs ->
         let new_acc =
-          if (deleted > since) && (created <= since)
+          if (deleted >= since) && (created <= since)
```

Comment:
Are you sure you still want to change this if you made Event.from behave the same?

Reply:
No. I don't want to make this change. Thought I'd removed that. That also affects my test results.